doc/conf.py

```diff
@@ -102,6 +102,15 @@ def _parse_skip_subdirs_file():
 # usage in the gallery.
 warnings.filterwarnings('error', append=True)
 
+# Prevent Sphinx-Gallery and jupytext incompatibility warning. This is spurious
+# and can be ignored for our use case. This has to be added before importing
+# jupyterlite_sphinx below, so that we can block the warning before it is raised.
+# TODO: Fix this in jupyterlite-sphinx
```

Comment:
Is there an issue you can link?

Reply:
Yes, there is one: https://github.com/jupyterlite/jupyterlite-sphinx/issues/258, I've added it in 86a5f7cd4e3d2bc81051e45a3390e728c95f21ac.